python/lsst/meas/algorithms/psfCandidate/__init__.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
```

Comment:
I've probably missed a few instances of this before, but we should probably be adding license statements to all these `__init__.py` files.